make-all.bash

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash -e
 
 # Build the syncthing library
-
-./make-go.bash arm
```

Comment:
Are the trust issues such that a binary download of https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz directly doesn't work? This is presumable the same place you download the source from anyway?